metric.graphqls

```diff
@@ -50,6 +50,8 @@ type KVInt {
     # 1. If ask for cpm metric, the unit and result should be count.
     # 2. If ask for response time (p99 or avg), the unit should be millisecond.
     value: Long!
+    # Since 9.5.0, isEmptyValue indicates whether value == 0 represents actually zero(false, default) or no data(true).
+    isEmptyValue: Boolean!
```

Comment:
Sorry for joining late.
Question: Does it make sense to **only** populate(set to true) when there is no data?

I guess that to make clients life easier it is better to always have that value present. I was just thinking about the payload increase in terms of size.

Reply:
You could only use it when the value is `0`. But from a server-side perspective, it must be returned from a GraphQL perspective. I am not sure what do you expect about `only`

Reply:
If the return `value: 0`  is not a real `0`, then set the `emptyValue=true`. Otherwise, when the value is a real `0` or `>0` don't even return the `emptyValue` property in the marshalled response.
But as I said, this could add unnecessary complexity at both server side and client side code.

Reply:
> If the return value: 0 is not a real 0, then set the emptyValue=true. Otherwise, when the value is a real 0 or >0 don't even return the emptyValue property in the marshalled response.

That may not be necessary, as the client does/will always require this field, as they don't know when it comes with 0. And this costs very little on the server side actually. :) We don't need to worry about that.